src/ui/components/shared/MaterialIcon.tsx

```diff
@@ -18,6 +18,22 @@ type MaterialIconProps = React.HTMLProps<HTMLDivElement> & {
   iconSize?: keyof typeof SIZE_STYLES;
 };
 
+let isChecking = false;
+function useMaterialIconCheck() {
+  useEffect(() => {
+    if (isChecking) {
+      return;
+    }
+    isChecking = true;
+    let id = setInterval(() => {
+      if (typeof document === "object" && (document as any).fonts.check("12px Material Icons")) {
+        document.body.classList.add("material-icon-loaded");
```

Comment:
Honestly, I'm terrified by this, but if there's no better way... 😬 
Have we considered `.ready()` API (https://stackoverflow.com/a/32292880/1895436), that resolves a Promise, instead of us checking on interval?

Maybe at least not inlining strings, but moving them to some `const`s would be better. (this suggestion is not a blocker)